manta-trusted-setup/src/groth16/test.rs

```diff
@@ -218,6 +222,17 @@ where
     );
 }
 
+/// Tests if bls13_381 pairing ratio is valid.
+#[test]
+fn has_valid_bls12_381_pairing_ratio() {
+    let mut rng = OsRng;
+    assert_valid_pairing_ratio::<Bls12_381>(
+        G1Affine::gen(&mut rng),
+        G2Affine::gen(&mut rng),
+        Fr::gen(&mut rng),
+    );
+}
+
```

Comment:
This is related to bls12-381 and not directly related to trusted-setup. Should we move this test to manta-crypto?

Reply:
The `manta-crypto` repo does not have any explicit curves yet, we will do that in another PR.